packages/terra-clinical-item-collection/tests/wdio/clinical-item-collection-spec.js

```diff
@@ -0,0 +1,92 @@
+/* eslint no-unused-expressions: 0 */
+/* global before, browser, Terra, expect */
+
+const viewports = Terra.viewports('tiny', 'small', 'medium', 'large', 'huge', 'enormous');
+
+describe('Item Collection', () => {
+  describe('Displays a item collection with all possibe elements', () => {
+    before(() => {
+      browser.url('/#/raw/tests/clinical-item-collection/item-collection-all-elements');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+  });
+
+  describe('Displays the correct collection when responding to breakpoint', () => {
+    before(() => {
+      browser.url('/#/raw/tests/clinical-item-collection/item-collection-all-elements');
+      browser.execute("document.getElementById('site').style.padding = 0;");
```

Comment:
Is it possible to add this line the the componentWillMount or componentDidMount methods of this component? It'd be great if the component contains all the screen setup, leaving this code only responsible for running screenshots.

Reply:
We can actually remove this now. If you check out the raw test urls, the padding is not longer applied to the pages.